src/helpers/documentation.js

```diff
@@ -235,9 +209,85 @@ export const buildDocsContents = (data, rootDir) => {
     insertPageData(relPath, contents, articles, rootPageData);
   });
 
+  // Now that content is in proper order, travel through tree and add Next Up link to each article
+  const contentsList = Object.values(contents);
+  contentsList.forEach((topicData, topicIndex) => {
+    const topicArticles = Object.values(topicData.articles);
+    const nextTopic = contentsList[topicIndex + 1];
+
+    addNextUpToArticles(topicArticles, 0, nextTopic);
+  });
   return contents;
 };
 
+/**
+ * addNextUpToArticles recursively travels through contents tree to add a Next Up property to each
+ * @param {array} articles List of articles of the current topic.
+ * @param {number} articleIndex The index of the article we're working with
+ * @param {number} nextTopic List of articles of the next topic
+ * @returns {void}
+ */
+const addNextUpToArticles = (articles, articleIndex, nextTopic) => {
```

Comment:
take each section of articles, and go through each one and add the next up link to each one.

if we encounter a nested group of articles, run through this func with that group